PumaGrasshopper/AttributeForm.cs

```diff
@@ -41,9 +174,10 @@ private void Cancel_Click(object sender, EventArgs e)
             Close();
         }
 
-        private void AttributeComboBox_SelectedIndexChanged(object sender, EventArgs e)
+        private void textBox1_TextChanged(object sender, EventArgs e)

```

Comment:
Consider renaming this in the designer so it's clear which text has changed here :)